### DISCUSSION TITLE
When is support for version 3.0/3.2 expected

### DISCUSSION THREAD

**Discussion by ErikNoort** [toltec-dev/toltec#673] [Support] [answered]
hi all,

Our CEO bought a nice Remarkable2, but never consulted with the IT team whether it supports encryption ;-)
He loves it, but we want him to stay secure, so we went looking for a solution and came across this excellent OSS initiative.
I came across the notice that anything beyond firmware 2.15 is not supported.
I was not able to determine how close (or far off) you are with adding support for 3.0/3.2 firmware.

Thanks in advance!

**Comment by Eeems** [comment, marked as answer]
There are a couple things holding us back:

- The `ddvk-hacks` package doesn't currently support 3.x, and that is closed source and waiting for the author to get around to it. They aren't really interested in 3.x yet though.
- There is a binding missing from Oxide's chroot of xochitl, not sure what it is yet, but that should be easy to fix when someone reports back: https://github.com/ddvk/remarkable2-framebuffer/issues/118
- None of the custom kernels or kernel modules have been tested on 3.x yet.
- None of the toltec maintainers have been running 3.x with toltec for any length of time to find any other issues yet. I usually run it for the week full support is in testing.

**Reply by jkesselm** [reply]
Any news? Any hope of news? The ReMarkable could make a good songbook, but PDFs indexed only by tagging (and no ability to bulk-edit the tags) are not a great user interface for that purpose. I need alternative apps...

I'm up to 3.9.3.1986 at this point. I'd consider rolling back to (and being stuck at) a supported version to get Toltec flexilbility, if that was safely doable, but I haven't seen any mention of whether that can be made to work.

**Reply by Eeems** [reply]
You are just asking this question everywhere you can, without waiting for a reply, aren't you? This is the third time I've seen you ask basically the same question in a different repository.

https://remarkable.guide/faqs.html#can-i-downgrade-to-a-different-os-version
#820

**Comment by paultopia** [comment]
Sorry, stupid question, but is the incompatibility just limited to the remarkable 2?  Or will remarkable 1 also be bricked if installed on OS 3.x?  Asking for a friend (me) before that friend (me) goes searching around to try to find a downgraded version of the OS...

**Reply by Eeems** [reply]
This should have the answer to your question: https://remarkable.guide/faqs.html#can-i-install-toltec-before-my-os-version-is-supported

**Reply by paultopia** [reply]
Thank you!

**Reply by grant-cox** [reply]
@Eeems, For anyone who has bought a new ReMarkable 2 and it shipped with 3.0+, do we simply need to wait? That is fine, but I want to ensure I'm not missing out on using at least a subset of the capabilities. For example, is it possible to install backend progs that do not need write access to the screen? I'd like to be able to rsync to the device, for example. Thanks!

**Reply by Eeems** [reply]
https://remarkable.guide/faqs.html#can-i-downgrade-to-a-different-os-version

**Reply by grant-cox** [reply]
Perfect. Just what I was looking for. Thank you.

**Comment by jkesselm** [comment]
I'm afraid so, sorry. It isn't obvious to an outsider how much crosstalk there is between the projects, and the answers could be different from different points of view.... I don't expect project X to answer for Y, thought if you can, great

--
   /_  Joe Kesselman (he/him/his)
-/ _) My Alexa skill for New Music/New Sounds fans:
   /   https://www.amazon.com/dp/B09WJ3H657/

Caveat: Opinionated old geezer with overcompensated writer's block. May be redundant, verbose, prolix, sesquipedalian, didactic, officious, or redundant.
________________________________
